modules/smmsBidAdapter.js

```diff
@@ -0,0 +1,160 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'smms';
+const ENDPOINT_BANNER = '//bidder.mediams.mb.softbank.jp/api/v1/prebid/banner';
+const ENDPOINT_NATIVE = '//bidder.mediams.mb.softbank.jp/api/v1/prebid/native';
+const COOKIE_SYNC_URL = '//bidder.mediams.mb.softbank.jp/api/v1/cookie/gen';
+const SUPPORTED_MEDIA_TYPES = ['banner', 'native'];
+const SUPPORTED_CURRENCIES = ['USD', 'JPY'];
+const DEFAULT_CURRENCY = 'JPY';
+const NET_REVENUE = true;
+
+function _encodeURIComponent(a) {
+  let b = window.encodeURIComponent(a);
+  b = b.replace(/'/g, '%27');
+  return b;
+}
+
+export function _getUrlVars(url) {
+  let hash;
+  const myJson = {};
+  const hashes = url.slice(url.indexOf('?') + 1).split('&');
+  for (let i = 0; i < hashes.length; i++) {
+    hash = hashes[i].split('=');
+    myJson[hash[0]] = hash[1];
+  }
+  return myJson;
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['smms'], // short code
```

Comment:
aliases is an optional property. Both your bidder code and alias used here are same. You can remove this if not needed

Reply:
thank you, I removed this.

Reply:
I removed this